zerver/webhooks/github/view.py

```diff
@@ -639,6 +639,10 @@ def is_commit_push_event(payload: WildValue) -> bool:
     return bool(re.match(r"^refs/heads/", payload["ref"].tame(check_string)))
 
 
+def is_merge_queue_push_event(payload: WildValue) -> bool:
+    return bool(re.match(r"^refs/heads/gh-readonly-queue/", payload["ref"].tame(check_string)))
```

Comment:
I know it’s copy+paste fallout, but this is a complicated way to spell `.startswith()`.

Reply:
@sbansal1999: Can you fix this, and the other occurrence in the file above it?

Reply:
Opened #25796.